contrib/cirrus/setup.sh

```diff
@@ -2,6 +2,8 @@
 
 set -xeo pipefail
 
+dnf up -y osbuild
```

Comment:
I would prefer to build new images instead of running on the fly updates that can fail if the repos are down or they are some new package conflicts, this was just to test since I asked Dusty about the CI issue and he mentioned we need to update osbuild.

Reply:
SGTM